components/coda/coda.app.mjs

```diff
@@ -1,11 +1,158 @@
+import axios from "axios";
```

Comment:
Prefer `@pipedream/platform` axios — see the docs from [this guideline](https://pipedream.com/docs/components/guidelines/#use-pipedream-platform-axios-for-all-http-requests)

Reply:
Issue addressed